components/Partners.js

```diff
@@ -22,7 +24,7 @@ function Partners({ partners }) {
           <div className="w-full rounded-lg border-2 shadow-lg hover:shadow-md my-5 hover:bg-gray-100 dark:bg-gray-900 dark:hover:bg-gray-800 dark:border-gray-800 dark:hover:border-gray-900 cursor-pointer ">
             <div className="flex md:flex-row flex-col">
               <img
```

Comment:
Hey @tanya17-05, can you please add the ID tag to this img tag with value darkMode.

Reply:
Sure, do I need to add id attribute to light mode logos as well??

Reply:
done @git-anurag-hub